internal/command/jsonformat/collections/slice.go

```diff
@@ -73,3 +82,27 @@ func ProcessSlice[Input any](before, after []Input, process ProcessIndices, isOb
 		}
 	}
 }
+
+// Returns if every item of before can be found in after
```

Comment:
nit: go style recommends that godoc for functions start with the name of the function eg.

```
// isReorder returns if ...
```